### DISCUSSION TITLE
Gocardless supported bank, not supported on Data Importer.

### DISCUSSION THREAD

**Discussion by Azokul** [firefly-iii/firefly-iii#8560] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Some EU banks (supported in Gocardless and Spectre natively) are not yet supported by Firefly importer 
![image](https://github.com/firefly-iii/firefly-iii/assets/10847617/c347033b-b7ef-49a2-90e8-c05e88f31178)

### Solution

Add support for EU banks via gocardless that doesn't use a divided checking / saving account. (Intesa San Paolo as example)

**Comment by JC5** [comment]
This is not a matter of EU bank support, but probably of missing asset accounts in Firefly III.

**Reply by Azokul** [reply]
Jeeeeeeeeeeeeeeeeeeeeeeeeeeez, what a dum dum. 
Thanks, you're the best.
